pkgs/development/libraries/tidyp/default.nix

```diff
@@ -1,14 +1,12 @@
-{ lib, stdenv, fetchFromGitHub }:
+{ lib, stdenv, fetchurl }:
 
 stdenv.mkDerivation rec {
   pname = "tidyp";
   version = "1.04";
 
-  src = fetchFromGitHub {
-    owner = "petdance";
-    repo = "tidyp";
-    rev = version;
-    sha256 = "0jslskziwzk4hb6i640fvpnbv2zxrvim6pdx2gwx5wyc64aviskc";
+  src = fetchurl {
+    url = "https://github.com/downloads/petdance/tidyp/${pname}-${version}.tar.gz";
```

Comment:
that URL is a 404 and only exists in cache.

Reply:
Hmm... it seems to work for me? 

```
grahamc@hyperchicken:~/projects/github.com/NixOS/nixpkgs/ > nix-build /nix/store/jg3qgrbmnm1vanljm2w2q30q48h84kg6-tidyp-1.04.tar.gz.drv --check
checking outputs of '/nix/store/jg3qgrbmnm1vanljm2w2q30q48h84kg6-tidyp-1.04.tar.gz.drv'...

trying http://tarballs.nixos.org/sha256/0f5ky0ih4vap9c6j312jn73vn8m2bj69pl2yd3a5nmv35k9zmc10
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  647k  100  647k    0     0   929k      0 --:--:-- --:--:-- --:--:--  929k
/nix/store/gbwl81l8rkypk47bi7y3bh70sk66rp0x-tidyp-1.04.tar.gz
```

Reply:
Oh I see.

Reply:
No, still seems fine:

```
grahamc@hyperchicken:~/projects/github.com/NixOS/nixpkgs/ > curl -L https://github.com/downloads/petdance/tidyp/tidyp-1.04.tar.gz --output foo
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   426  100   426    0     0    460      0 --:--:-- --:--:-- --:--:--   460
100  647k  100  647k    0     0   464k      0  0:00:01  0:00:01 --:--:-- 2900k
grahamc@hyperchicken:~/projects/github.com/NixOS/nixpkgs/ > sha256sum foo
20b0fad32c63575bd4685ed09b8c5ca222bbc7b15284210d4b576d0223f0b338  foo


grahamc@hyperchicken:~/projects/github.com/NixOS/nixpkgs/ > sha256sum /nix/store/gbwl81l8rkypk47bi7y3bh70sk66rp0x-tidyp-1.04.tar.gz
20b0fad32c63575bd4685ed09b8c5ca222bbc7b15284210d4b576d0223f0b338  /nix/store/gbwl81l8rkypk47bi7y3bh70sk66rp0x-tidyp-1.04.tar.gz

```

Reply:
That is a very legacy download page. The listing is under https://github.com/petdance/tidyp/downloads. Lets add a comment that we do not break this in the future.